### PR TITLE
Add fhicl files for production (PDS)

### PR DESCRIPTION
**Pull request by fjnicolas**
This PR adds fhicl files for MC Production (PDS deconvolution testing samples).

### CODE REVIEW

File: sbndcode/OpDetReco/OpDeconvolution/reco_deco_sbnd/standard_reco1_deco_savewf_sbnd.fcl [hunk 1]
```diff
@@ -12,10 +12,13 @@ physics.reco1: [opdecopmt, @sequence::physics.reco1]
...
          "drop *raw::RawDigits*_*_*_*",
          #"drop sim::SimChannels_*_*_*",
          "drop sim::OpDetBacktrackerRecords_*_*_*",
          "drop *_linecluster_*_*",
          "drop *_fasthit_*_*"
-     ]
\ No newline at end of file
+     ]
+
+physics.producers.opdecopmt.OpDecoAlg.OpDetDataFile: "OpDetSim/digi_pmt_sbnd_v2int0.root"
```
Comment (jzennamo) [inline sbndcode/OpDetReco/OpDeconvolution/reco_deco_sbnd/standard_reco1_deco_savewf_sbnd.fcl:23]: Where does this file live? sbndata?

**Comment by jzennamo** [comment]
trigger build

**Comment by jzennamo** [comment]
@fjnicolas this looks great! Could you include a reference to how this new workflow changes the expected physics results and how it changes the run time, memory, and file sizes? Thanks!

File: sbndcode/OpDetReco/OpDeconvolution/reco_deco_sbnd/standard_reco1_deco_savewf_sbnd.fcl (hunk 1 above)
Comment (fjnicolas) [reply sbndcode/OpDetReco/OpDeconvolution/reco_deco_sbnd/standard_reco1_deco_savewf_sbnd.fcl:23]: Yes, it's in sbnd_data from v01_08_00. It contains a modified SER for the PMTs.

**Comment by fjnicolas** [comment]
> @fjnicolas this looks great! Could you include a reference to how this new workflow changes the expected physics results and how it changes the run time, memory, and file sizes? Thanks!

- I don't expect run time, memory and file size to change for DetSim stage. It completely changes the PMT output waveforms (switch to bipolar/with overshoot signals, aka test bench SER).
- For Reco1 it takes ~5 extra seconds to run the `opdecopmt` stage. Memory usage is similar for both workflows. Below the processing summary for 5 nu+cosmic events on gpvms.

Run time and memory usage for modified workflow including PMT deconvolution (only showing PDS related processes):
numucosmic_Reco1_PMTdeco copia
numucosmic_Reco1_PMTdeco copia 2

For standard Reco1:
numucosmic_Reco1_standard copia
numucosmic_Reco1_standard copia 2

Reco1 file sizes:
- Deconvolution:  226 MB
- Standard: 158 MB

NB: file sizes when running `standard_reco1_deco_savewf_sbnd.fcl` which saves both raw and deconvolved waveforms (for debugging purposes with the MC test sample). If we don't save the waveforms I don't expect the file sizes to change significantly.

Previous studies showed a PE recovering resolution ~5% and unbiased at the level of 5% at the OpHit level. A summary can be found in [docdb-22541](https://sbn-docdb.fnal.gov/cgi-bin/sso/ShowDocument?docid=22541).

**Review by pgreen135 — APPROVED** [review]

**Pull request merged**